packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -414,6 +373,89 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
   }
 }
 
+class InspectorSummaryTreeControls extends StatelessWidget {
+  const InspectorSummaryTreeControls({
+    Key key,
+    @required this.constraints,
+    @required this.isSearchVisible,
+    @required this.onRefreshInspectorPressed,
+    @required this.onSearchVisibleToggle,
+    @required this.searchFieldBuilder,
+  }) : super(key: key);
+
+  static const _searchBreakpoint = 375.0;
+
+  final bool isSearchVisible;
+  final BoxConstraints constraints;
+  final VoidCallback onRefreshInspectorPressed;
+  final VoidCallback onSearchVisibleToggle;
+  final Widget Function() searchFieldBuilder;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        _controlsContainer(
+          context,
+          Row(
+            children: <Widget>[
+              const SizedBox(width: denseSpacing),
+              const Text('Widget Tree'),
+              const SizedBox(width: denseSpacing),
```

Comment:
nit: just wrap `Text('WidgetTree')` in a padding widget with horizontal padding `denseSpacing`